website/documentation/guides/security-and-compliance/kubernetes-hardening/_index.md

```diff
@@ -31,6 +31,13 @@ The Gardener team takes security seriously, which is why we mandate the Security
 |242394|Kubernetes Worker Nodes must not have the sshd service enabled| :x: | Enabled to allow debugging of network issues, but it is possible to deactivate via the [sshAccess](https://gardener.cloud/docs/gardener/api-reference/core/#core.gardener.cloud/v1beta1.SSHAccess) setting|
 |242434|Kubernetes Kubelet must enable kernel protection|:white_check_mark:|Enabled for Kubernetes v1.26 or later unless disabled explicitly via [protectKernalDefaults](https://gardener.cloud/docs/gardener/api-reference/core/#kubeletconfig)|
 |245541|Kubernetes Kubelet must not disable timeouts|:white_check_mark:| Enabled for Kubernetes v1.26 or later unless disabled explicitly via [streamingConnectionIdleTimeout](https://gardener.cloud/docs/gardener/api-reference/core/#kubeletconfig)|
+|242402|The Kubernetes API Server must have an audit log path set| :x: | It is the user's responsibility to configure an audit extension that meets the requirements of their organization. Depending on the audit extension implementation the audit logs do not always need to be written on the filesystem, i.e. when `--audit-webhook-config-file` is set and logs are sent to an audit backend.|
```

Comment:
Shall we introduce a dedicated section `Audit Log Configurations` or we better keep the rules in `Control Plane Configuration`?

Reply:
I also thought about this, but I do not have strong opinion on the matter. Do you prefer for them to be separate?

Reply:
I have a slight preference to keep them separated.